src/xdist/remote.py

```diff
@@ -56,6 +59,45 @@ def worker_title(title):
         pass
 
 
+class RemoteMessageHandler(logging.Handler):
+    """
+    This handler sends events to a queue. Typically, it would be used together
+    with a multiprocessing Queue to centralise logging to file in one process
+    (in a multi-process application), so as to avoid file write contention
+    between processes.
+
+    This code is new in Python 3.2, but this class can be copy pasted into
```

Comment:
Seems like this was copied from somewhere, can you add a link to where it was copied from?